prow/cmd/webhook-server/main.go

```diff
@@ -223,15 +226,22 @@ func handleSecrets(client ClientInterface, ctx context.Context, clientoptions cl
 		cert = secretsMap[certFile]
 		privKey = secretsMap[privKeyFile]
 	}
-
+	logrus.Info(caPem)
+	err = reconcileWebhooks(ctx, caPem, cl)
+	if err != nil {
+		return "", "", err
+	}
```

Comment:
this should be after `isCertValid` function, and `isCertValid` check should actually happen as a `else if` condition from the if/else check above